hasktorch/src/Torch/TensorFactories.hs

```diff
@@ -54,6 +54,18 @@ linspace start end steps opts = unsafePerformIO $ (cast4 LibTorch.linspace_sslo)
 logspace :: (Scalar a, Scalar b) => a -> b -> Int -> Double -> TensorOptions -> Tensor
 logspace start end steps base opts = unsafePerformIO $ (cast5 LibTorch.logspace_ssldo) start end steps base opts
 
+empty :: [Int] -> TensorOptions -> Tensor
+empty = mkFactoryUnsafe LibTorch.empty_lo
```

Comment:
I don't think we should expose this one, because it's not fully referentially transparent (contents of the tensor depend on the memory we get from the allocator). It's fine to use it in the internals for performance reasons, when you know you'll be immediately writing to that tensor, but I don't think this should be part of the public API for now.

Reply:
Okay, will leave it out of the public interface, maybe put it in an internal module or just delete it for now.